index.js

```diff
@@ -476,12 +481,12 @@ class Plugin extends BtpPlugin {
           })
         })
         .catch((e) => {
-          debug('fund tx/notify failed:', e)
+          this._log.error('fund tx/notify failed:', e)
           this._funding = false
         })
     }
 
-    debug('setting new claim. amount=' + amount)
+    this._log.info('setting new claim. amount=' + amount)
```

Comment:
this will happen on almost every packet so we can probably put it on the debug level

Reply:
This seems trivial enough to me that it could go into trace, especially if it may spam over more valuable debug output. Thoughts?

Reply:
sure, trace sounds ok